examples/axios/pages/api/data.js

```diff
@@ -7,7 +7,7 @@ const projects = [
 export default (req, res) => {
   if (req.query.id) {
     // a slow endpoint for getting repo data
-    fetch(`https://api.github.com/repos/${req.query.id}`)
+    axios(`https://api.github.com/repos/${req.query.id}`)
       .then(resp => resp.json())
```

Comment:
Does axios support this?

Reply:
Yes, for a standard 'GET' request. I'll update the Typescript example as well.

Reply:
It seems `axios` doesn't support `.json()`:

![image](https://user-images.githubusercontent.com/3676859/90759815-8e546200-e313-11ea-80b2-d14edd00ad6f.png)

https://runkit.com/shuding/5f3e51784380ec001acd0501

Reply:
You're correct. The response requires the `.data` object for axios. I've updated the other example in a separate PR and can close this one.